app/src/main/java/org/wikipedia/analytics/Funnel.kt

```diff
@@ -16,8 +16,6 @@ abstract class Funnel @JvmOverloads internal constructor(protected val app: Wiki
                                                          private val revision: Int, private val sampleRate: Int = SAMPLE_LOG_ALL,
                                                          private val wiki: WikiSite? = null) {
 
-    private val sampleRateRemoteParamName = schemaName + "_rate"
```

Comment:
(this dynamically-programmable sample rate was never used in RemoteConfig, and never will be.)